recompose-flow.md

```diff
@@ -31,16 +31,16 @@ const enhancer2: HOC<*, EnhancedCompProps> = compose(
 )
 ```
 
-I was able to declare just Enhanced component props type definition, all types were infered properly and error was readable.
+I was able to declare just the enhanced component props type definition; then all types were inferred properly, and the error was readable.
 
-It was clear that if it would be possible to make same idea to work for most recompose enhancers there will be really easy to work with flow and recompose.
+It was clear that if it were possible to make the same idea work for most Recompose enhancers, it would be easy to work with Flow and Recompose.
 
-And thanks to flow type inference and @GiulioCanti, for most enhancers it already worked.
+And thanks to Flow type inference and @GiulioCanti, for most enhancers it already worked.
```

Comment:
For this second instance of @GiulioCanti, my preference would be to change it to just "Giulio" or "Giulio Canti". However, if you're writing for a community which regularly uses @username to refer to people (even in blog posts (that is, outside the context of social media and collaboration websites)), then you can keep this. I would have a look at the other posts on the Flow Medium blog and see what they usually do.